prettify_results.py

```diff
@@ -234,24 +234,15 @@ def results_to_map(results_file, stats_files):
 if __name__ == "__main__":
     PARSER = ArgumentParser()
     PARSER.add_argument("-r", "--resultsFile", dest="results_file",\
-        default=False, help="File with results of SourcererCC (results.pairs).")
+        required=True, help="File with results of SourcererCC (results.pairs).")
     PARSER.add_argument("-s", "--statsFiles", dest="stats_files",\
-        default=False, help="File or folder with stats files (*.stats).")
+        required=True, help="File or folder with stats files (*.stats).")
 
     if len(sys.argv) == 1:
         print("No arguments were passed. Try running with '--help'.")
         sys.exit(0)
 
     OPTIONS = PARSER.parse_args(sys.argv[1:])
```

Comment:
I would suggest to use `OPTIONS = PARSER.parse_args()` - so you can skip check of `sys.argv` because `parse_args` will raise an exception if not all required arguments provided.